lib/matplotlib/text.py

```diff
@@ -309,6 +315,27 @@ def get_rotation_mode(self):
         """Return the text rotation mode."""
         return self._rotation_mode
 
+    def set_antialiased(self, b):
```

Comment:
```suggestion
    def set_antialiased(self, antialiased):
```
For consistency with most other setters (although it rarely matters here...).